zds/searchv2/views.py

```diff
@@ -149,19 +149,18 @@ def get_queryset(self):
             scored_queryset = FunctionScore(query=queryset, boost_mode='multiply', functions=weight_functions)
             search_queryset = search_queryset.query(scored_queryset)
 
-            # highlighting:
+            # Highlighting:
             search_queryset = search_queryset.highlight_options(
                 fragment_size=150, number_of_fragments=5, pre_tags=['[hl]'], post_tags=['[/hl]'])
             search_queryset = search_queryset.highlight('text').highlight('text_html')
 
-            # executing:
+            # Executing:
             return self.index_manager.setup_search(search_queryset)
 
         return []
 
     def get_queryset_publishedcontents(self):
-        """Find in PublishedContents.
-        """
+        """Searches in PublishedContents."""
```

Comment:
Il y a un truc qui me gène ici, c'est que "PublishedContents" est formaté comme un nom de classe alors que la classe est au singulier. Je pense que le plus correct serait "in published contents" ou "in PublishedContent objects". ;)